open_discussions/templates/base.html

```diff
@@ -10,15 +10,11 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
     <link rel="stylesheet" type="text/css" href="{% static 'hijack/hijack-styles.css' %}" />
     <link rel="icon" href="{% static 'images/favicon.ico' %}" />
-    {% if js_settings_json %}
-    <script type="text/javascript">
-    var SETTINGS = {{ js_settings_json|safe }};
-    </script>
-    {% render_bundle 'style' %}
-    {% endif %}
+
     <title>{% block title %}{% trans "Open Discussions" %}{% endblock %}</title>
     <meta name="description" content="{% block description %}{% endblock %}">
     <meta name="keywords" content="{% block keywords %}{% endblock %}">
+    {% render_bundle 'style' %}
```

Comment:
On this branch but not master I'm seeing this error in the console:

<img width="431" alt="Screen Shot 2020-07-30 at 10 13 45 AM" src="https://user-images.githubusercontent.com/187676/88933952-0f0cca80-d24e-11ea-9573-81db195a8d89.png">